packages/editor/src/panels/properties/common/NodeEditor.tsx

```diff
@@ -82,6 +83,11 @@ const NodeEditor = ({
 }: ComponentDropdownProps & EditorPropType) => {
   const { t } = useTranslation()
 
+  const minimizedByDefault = false
+
+  //ensure the entry exists in the state but do not modify it if it already exists
+  if (name) ComponentDropdownStateFunctions.ensureInitializedComponentState(entity, name, minimizedByDefault)
```

Comment:
state mutations should be inside a `useEffect`. We can add a `useImmediateEffect` if it throws errors with just a `useEffect`